src/components/video.tsx

```diff
@@ -28,7 +34,8 @@ export class Video extends React.Component<IVideoProps, any> {
     return (
       <div className="video" style={{backgroundImage, ...style}}>
         <div className="details">
-          <div className="name">
+          <div className="name"
+            onClick={this.onClickTitle.bind(this)}>
```

Comment:
If you will use arrow function, you can change this to

```suggestion
            onClick={this.onClickTitle}>
```